easypost/easypost_object.py

```diff
@@ -62,7 +63,7 @@ def convert_to_easypost_object(response, api_key, parent=None, name=None):
     }
 
     if isinstance(response, list):
-        return [convert_to_easypost_object(r, api_key, parent) for r in response]
+        return [convert_to_easypost_object(response=r, api_key=api_key, parent=parent) for r in response]
```

Comment:
I know you didn't write it, but can we please define `r` here? It's better to be explicit than implicit and we can clean this up a bit since we are here.